### ISSUE TITLE
[v12] GUI for relations

### ISSUE THREAD

**Issue by DavidBruchmann** [FriendsOfTYPO3/extension_builder#748]
**Is your feature request related to a problem? Please describe.**
the relations label is displayed differently than other properties and has confusing padding in the parent window

![image](https://github.com/FriendsOfTYPO3/extension_builder/assets/999146/0a9f53e7-8ee2-46f6-9cbc-70b1bee6fbb5)

**Describe the solution you'd like**
I think it should be shown like the properties option and grayed out if empty.
The green button to create new relations should be shown when the option is opened by the toggle.

**Comment by PKuhlmay** [comment]
You are Right. The GUI still needs some CSS improvements. The Relations area is no accordion because this results into other issues. I could add paddings and some other styling, so it looks like an accordion bur technically it isn’t an accordion which you can toggle.

**Comment by PKuhlmay** [comment]
This issue was filed against the old React JS GUI (branch `dev-typo3-12-reactjs`), which has been completely replaced by a LIT-based GUI since v12. If this problem still exists in the current LIT GUI, please open a new issue referencing this one.
